sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/AnomalyDetection/ChangeThresholdCondition.cs

```diff
@@ -1,25 +1,50 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
 using Azure.Core;
 
 namespace Azure.AI.MetricsAdvisor.Models
 {
+    // TODODOCS: whole class is hard to explain.
+
     /// <summary>
+    /// A condition used for anomaly detection. Using the value of a previously ingested data
+    /// point as reference, sets bounds to specify the range in which data points are expected
+    /// to be. Points with unexpected values will be considered an anomaly according to the
+    /// rules set by the <see cref="SuppressCondition"/>.
     /// </summary>
     public partial class ChangeThresholdCondition
     {
         /// <summary>
+        /// Creates a new instance of the <see cref="ChangeThresholdCondition"/> class.
         /// </summary>
-        public AnomalyDetectorDirection AnomalyDetectorDirection { get; }
+        /// <param name="changePercentage">The relative change </param>
+        /// <param name="shiftPoint">When set to N, sets as reference the data point that's N positions before the current point. Value must be at least 1.</param>
+        /// <param name="isWithinRange"></param>
+        /// <param name="anomalyDetectorDirection"></param>
+        /// <param name="suppressCondition">Used to avoid outright labeling every single unexpected data point as an anomaly.</param>
```

Comment:
`SuppressCondition` is probably the hardest to explain. If you read the whole class description, does it sound clear to you?
```cs
    /// <summary>
    /// Used to avoid outright labeling every single unexpected data point as an anomaly. An
    /// anomaly will be detected only when the latest ingested points contain a minimum
    /// threshold of unexpected values, indicating that there's in fact something wrong with
    /// the data.
    /// </summary>
```

Reply:
I'm still not quite sure. Let me see if I can re-phrase it and validate my understanding.

In the absence of this setting, every anomalous data point is labeled as such. With this setting, a set of ingested data points will only be labeled if the number of anomalous values exceeds a defined threshold.

Reply:
Not exactly, but quite close.

It doesn't label "whole sets" of data points as anomalies, but only one point at a time. Every time a time series gets a new data point, the service takes a look at the latest points for that series. If the amount of "unexpected" points exceeds a user-specified threshold, then **only the latest** point that caused this will be labeled as an anomaly.

There's no way to make this setting absent, though. It's required. If you want to make every "unexpected" point an anomaly, you could make:
```cs
suppressCondition = new SuppressCondition(minimumNumber: 1, minimumRatio: 100); // minimumNumber should be renamed, probably.
```

Reply:
> Every time a time series gets a new data point, the service takes a look at the latest points for that series. If the amount of "unexpected" points exceeds a user-specified threshold, then **only the latest** point that caused this will be labeled as an anomaly.

👍 This is great!